### DISCUSSION TITLE
contributing.md update

### DISCUSSION THREAD

**Discussion by n0n3br** [greatbuilderhq/greatbuilder#22] [Ideas]
- Update Contribute to Code
  - Help improve our docs, including this one!
  - Check open isssues, we might have a problem that needs help
  - Create new components ~~from Langchain~~
  - Update existing components such as extending functionality, fixing bugs
  - ~~Add new build ideas~~
- Remove Developers section (we don't have these modules)
- Update For Development
  - Create `.env` file and specify the values (refer to `.env.example`) ~~in `packages/ui`~~
  - ~~Create `.env` file and specify the `PORT` (refer to `.env.example`) in `packages/server`~~
- Replace env variables with the list below (auth variables are for nextauth)
  - PORT=3000
  - DATABASE_HOST
  - DATABASE_PORT
  - DATABASE_USER
  - DATABASE_PASSWORD	
  - DATABASE_NAME
  - AUTH_SECRET
  - AUTH_AUTH0_ID
  - AUTH_AUTH0_SECRET
  - AUTH_AUTH0_ISSUER
  - AUTH_FACEBOOK_ID
  - AUTH_FACEBOOK_SECRET
  - AUTH_GITHUB_ID
  - AUTH_GITHUB_SECRET
  - AUTH_GOOGLE_ID
  - AUTH_GOOGLE_SECRET
  - AUTH_TWITTER_ID
  - AUTH_TWITTER_SECRET
  - AUTH_APPLE_ID
  - AUTH_APPLE_SECRET

**Comment by TameemAsim** [comment]
@n0n3br Kindly tell me what is greatbuilder

**Reply by n0n3br** [reply]
Maybe we should ask @greatbuilderhq what's the final goal. I'm trying to contribute but still don't know where it's going

**Comment by TameemAsim** [comment]
> Maybe we should ask @greatbuilderhq what's the final goal. I'm trying to contribute but still don't know where it's going

I agree.
@greatbuilderhq Kindly tell us
